source/codegen/metadata/nidaqmx/functions_addon.py

```diff
@@ -1,4 +1,139 @@
 functions_override_metadata = {
+    'ReadAnalogF64': {
+        'python_codegen_method': 'CustomCode_Read',
+    },
+    'ReadAnalogScalarF64': {
+        'python_codegen_method': 'CustomCode_Read_Scalar',
+    },
```

Comment:
This feature is not Python-specific. nidaqmx-python is the first DAQmx gRPC client to use these additional gRPC metadata keys, but other DAQmx gRPC clients will also benefit.

Reply:
Removed these additional metadata.